### ISSUE TITLE
Regression seen in 3.0.3 with Update Credentials

### ISSUE THREAD

**Issue by JillieBeanSim** [zowe/zowe-explorer-vscode#3339]
**Describe the bug**

reported in OMP ZE slack channel and confirmed with testing, happens inconsistently

**To Reproduce**

1. Right click on Profile name, select manage profile and update credentials
2. enter update of password
3. get confirmation of successful update
4. See error for invalid creds

**Expected behavior**

update should successfully update cred manager and what is used in ZE without issue

**Screenshots**

**Desktop (please complete the following information):**

- OS:
- Zowe Explorer Version:
- (Optional) Zowe CLI Version:
- (Optional) Are you using Secure Credential Store?

**Additional context**

**Comment by traeok** [comment]
Closing as not planned for the time being, since the user who initially encountered the issue can no longer reproduce the bug. Will reopen if reproducible.

**Comment by davenice** [comment]
Just to add a couple more snippets in case this comes up again.

I'm on Mac, using Zowe Explorer 3.0.3.

Every time i right-clicked, did manage profile, update credentials, i'd see this error message:
`"Update Credentials" operation not supported when "autoStore" is false`

I had autoStore set once in my Zowe config file, to true.

I checked in my Mac keychain and removed all of the Zowe entries from the login keychain.

Still nothing. I then left it for a few weeks - when Trae came back to me I attempted to recreate while debugging in Zowe Explorer and this time the authentication happened and I can see the keychain entries have been created correctly.
